### PR TITLE
adding `navigatorDDG` configuration

### PR DESCRIPTION
**Pull request by shakyShane**
This PR is here to open the discussion about the proposed naming of the new DOM API that we're adding

Please see the following task: https://app.asana.com/0/72649045549333/1201487308673313/f
Tech Design: https://app.asana.com/0/0/1201508656742121/f

In the tech design, I suggested 2 names

```
navigatorNamespace
navigatorDDG
```

I don't have a strong preference, and I'm open to hearing other idea too?

### CODE REVIEW

**Comment by shakyShane** [comment]
@jonathanKingston Only enabling this on extension,ios,macos initially.

**Comment by jonathanKingston** [comment]
@shakyShane this seems good to me, is it ready for review now? Rebase is needed, "update branch" should work also.

**Comment by shakyShane** [comment]
@jonathanKingston rebased 👍🏻

**Comment by shakyShane** [comment]
@jonathanKingston updated to `navigatorDDG`

File: features/navigator-d-d-g.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jonathanKingston) [inline features/navigator-d-d-g.json:1]: This file name makes me want to cry.

@SlayterDev I think we should change this to `navigatorInterface` (interface is the lingo specifications use to describe these kinda things)

Comment (shakyShane) [reply features/navigator-d-d-g.json:1]: @jonathanKingston I'll update the PR, I think `navigatorInterface` is better anyway, regardless of weird filenames

**Comment by shakyShane** [comment]
@jonathanKingston what are the final steps for getting this released?

**Review by jonathanKingston — APPROVED** [review]

**Comment by jonathanKingston** [comment]
None, thanks for bearing with all the feedback!

**Pull request merged**